cmd/sendsignal.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"github.com/urfave/cli/v2"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/apps"
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/Scalingo/cli/detect"
+	"github.com/Scalingo/go-utils/errors"
+)
+
+var (
+	sendSignalCommand = cli.Command{
+		Name:     "send-signal",
+		Category: "App Management",
+		Usage:    "Send SIGUSR1/2 to your application containers",
```

Comment:
nitpick: just a suggestion, don't know what would be the best
```suggestion
		Usage:    "Send SIGUSR1 or SIGUSR2 to your application containers",
```